README.md

```diff
@@ -4,46 +4,24 @@ An open source book about design patterns and idioms in the Rust programming
 language that you can read [here](https://rust-unofficial.github.io/patterns/).
 
 
-## Contents
```

Comment:
Maybe leaving this and linking to the SUMMARY.md could be nice?

Reply:
mmm..Why the user might want to browse the book from github and not from the online version of the book?
It might be confusing for them

Reply:
Yeah, on first glance I don't have anything in mind as well. And we are always able to add that. So sure